### PR TITLE
hashId server change

### PR DESCRIPTION
**Pull request by henryzbw**
HashId change on server side

### CODE REVIEW

**Comment by henryzbw** [comment]
#77

**Comment by henryzbw** [comment]
@dwalintukan I don't know how to add the hashId in translate to make it work. since i need `this.hashId = `, but i don't know where can i get it to here. i tried to add `this.hashId = hashId`, but the hashId is undefined here. so it caused error.

**Comment by dwalintukan** [comment]
@henryzbw I don't mean to actually add the hashId in there, but you notice how like `name` and `options` for Oracle are null? I meant like a placeholder.

File: src/api/address-manager.js [hunk 1]
```diff
@@ -11,8 +11,7 @@ function getContract() {
 
 const AddressManager = {
   async eventEscrowAmount(args) {
-    const {
-      senderAddress, // address
+    const { senderAddress, // address
```
Comment (dwalintukan) [inline src/api/address-manager.js]: hm weird linter change. will have to change the eslint rules later.

**Pull request merged**